packages/pn-personafisica-webapp/src/redux/notification/actions.ts

```diff
@@ -275,7 +275,7 @@ export const exchangeNotificationQrCode = createAsyncThunk<BffCheckAarResponse,
       const response = await notificationReceivedApiFactory.checkAarQrCodeV1(params);
       return response.data;
     } catch (e: any) {
-      return rejectWithValue(parseError(e));
```

Comment:
Restore the old version and use the AppResponsePublisher to intercept the error and manage it properly. Do the same for PG.

Reply:
AppResponsePublisher can be useful when we have to manage the error in a way that is different from the standard way (i.e. show the toast error).
In this case, it's ok to show the error but we have to show it with a different message. So ask to Daniela or Norma the copy (tag them in the issue) and after manage the error through the AppError